hugegraph-core/src/main/java/com/baidu/hugegraph/util/ConfigUtil.java

```diff
@@ -50,24 +63,36 @@
     private static final String CHARSET = "UTF-8";
 
     public static void checkGremlinConfig(String conf) {
-        YamlConfiguration yamlConfig = new YamlConfiguration();
+        Parameters params = new Parameters();
         try {
-            yamlConfig.load(conf);
+
+            FileBasedConfigurationBuilder<FileBasedConfiguration> builder =
+                new FileBasedConfigurationBuilder(YAMLConfiguration.class)
+                    .configure(params.fileBased().setFileName(conf));
+            YAMLConfiguration config = (YAMLConfiguration) builder
+                                                           .getConfiguration();
+
+            List<HierarchicalConfiguration<ImmutableNode>> nodes =
+                                           config.childConfigurationsAt(
+                                           NODE_GRAPHS);
+            E.checkArgument(nodes == null || nodes.isEmpty()
+                            || nodes.size() == 1,
+              "Not allowed to specify multiple '%s' nodes in " +
+              "config file '%s'", NODE_GRAPHS, conf);
+
+            ImmutableNode root = null;
+            NodeModel<ImmutableNode> nodeModel = null;
+            NodeHandler<ImmutableNode> nodeHandler = null;
+            for (HierarchicalConfiguration<ImmutableNode> node : nodes) {
+                E.checkArgument((nodeModel = node.getNodeModel()) != null
+                  && (nodeHandler = nodeModel.getNodeHandler()) != null
```

Comment:
our style requires the symbol to be attached to the previous line, like "+" and "&&" symbols